packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -621,15 +628,42 @@ class StackViewport extends Viewport implements IStackViewport {
   }
 
   /**
-   * Sets the properties for the viewport on the default actor. Properties include
-   * setting the VOI, inverting the colors and setting the interpolation type, rotation
+   * Update the default properties of the viewport and add properties by imageId if specified
+   * @param ViewportProperties - The properties to set
+   * @param imageIdIndex If given, we set the default properties only for this image index
+   */
+  public setDefaultProperties(
+    ViewportProperties: StackViewportProperties = {},
```

Comment:
it doesn't make sense to default it to {}, if someone wants to setDefaultProperties they should provide one

Reply:
also seems like my previous comments have not been replied. Why we are doing this based on imageIdIndex and not imageId?

Reply:
Yes it was a mistake for sure, I am updating with ImageId now